contracts/multisig/src/state.rs

```diff
@@ -19,3 +20,5 @@ pub fn get_key(store: &dyn Storage, key_id: &KeyID) -> Result<Key, ContractError
             key_id: key_id.to_string(),
         })
 }
+
+pub const PUB_KEYS: Map<(Addr, KeyType), PublicKey> = Map::new("registered_pub_keys");
```

Comment:
add a comment that the key type is part of the key so that signers can register multiple keys with different types